apps/studio/Dockerfile

```diff
@@ -9,10 +9,10 @@ RUN apk add --no-cache libc6-compat
 RUN apk update
 # Set working directory
 WORKDIR /app
-RUN npm install --global turbo
+RUN npm install --global turbo@1
```

Comment:
Need to pin the version because the config files here are not compatible with Turbo v2.